cmd/clusterctl/hack/local-overrides.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+
+# Copyright 2015 The Kubernetes Authors.
```

Comment:
Did this come from some other file that was written in 2015 or should this be 2020?

Reply:
copy paste, sorry